README.md

```diff
@@ -242,4 +248,4 @@ All commit message must conform to [Conventional Commit Messages](https://www.co
 | `feat(pencil): add 'graphiteWidth' option` | ~~Minor~~ Feature Release |
 | `perf(pencil): remove graphiteWidth option`<br/><br/>`BREAKING CHANGE: The graphiteWidth option has been removed.`<br/>`The default graphite width of 10mm is always used for performance reasons.` | ~~Major~~ Breaking Release |
 
-When opening PRs commit messages are checked using commitlint.
\ No newline at end of file
+When opening PRs commit messages are checked using commitlint.
```

Comment:
This change (the removal of the blank newline) was also unintentional. These are the hazards of using GitHub's web UI for editing files ;)